keel-core/src/main/kotlin/com/netflix/spinnaker/keel/rest/AuthorizationSupport.kt

```diff
@@ -196,4 +202,19 @@ class AuthorizationSupport(
     }
 
   private fun Boolean.toAuthorization() = if (this) "ALLOWED" else "DENIED"
+
+  suspend fun <R> withSpinnakerAuthHeaders(
```

Comment:
This is the helper function to include the user and account headers into the coroutine context so it can be picked up by the custom retrofit call adapter.